Week3/homework/step2-1.js

```diff
@@ -1,9 +1,7 @@
 'use strict';
 
 function foo(func) {
-  // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
+  return func();
```

Comment:
Good! No need to return though because you're not doing anything with the result of calling `foo`

Reply:
Ok. I understand.